### DISCUSSION TITLE
Puzzle 41 Associate IDs

### DISCUSSION THREAD

**Discussion by maximilian22x** [smpetersgithub/AdvancedSQLPuzzles#26] [General]
https://github.com/smpetersgithub/AdvancedSQLPuzzles/blob/2306e23e83580f16670014850f061a6ab73e3c4f/Advanced%20SQL%20Puzzles/Advanced%20SQL%20Puzzles%20Solutions.sql#L1950

What is the column "**grouping**" for? Do not understand that column.
Why this code is not enough?
```
SELECT Associate1 
FROM #Associates 
Union 
SELECT Associate2 
FROM #Associates 
```
![image](https://user-images.githubusercontent.com/82707974/205120683-284ed422-f7ee-4804-90db-50878870788f.png)

**Comment by smpetersgithub** [comment]
The puzzle needs to be better written.

Here is a better description.

_The following table shows two hierarchical structures. 
1) The first is the association between Anne, Betty, Charles, Dan and Emma.
2) The second is the association between Francis, George, and Harriet.

Write an SQL statement that creates a grouping number for each hierarchical association and display the member in the associations._

**Comment by smpetersgithub** [comment]
Closing
